### ISSUE TITLE
Bring back SM tools and package exporter

### ISSUE THREAD

**Issue by 1033Forest** [stepmania/stepmania#1194]
Should we bring this program back? The tool that helped us install SM songs and add-ons, and helped with other things. This way we can easily install .smzip files again.

**Comment by shakesoda** [comment]
Not really, although a rewrite of it has been discussed so it'd be cross platform.

You can install .smzip files by putting them in the Packages folder and the game can also handle them directly.
